Capri.Web/Configuration/Mapper/ProposalMappingProfile.cs

```diff
@@ -15,6 +15,26 @@ public ProposalMappingProfile()
             .ForMember(
                 view=>view.Students, 
                 o=>o.MapFrom(proposal=>proposal.Students.Select(s=>s.Id)));
+
+            CreateMap<Proposal, ProposalCsv>()
+            .ForMember(csv => csv.Promoter, o => o.MapFrom(p => GetFullNameFor(p.Promoter)))
+            .ForMember(csv => csv.Course, o => o.MapFrom(p => p.Course.Name))
+            .ForMember(csv => csv.Faculty, o => o.MapFrom(p => p.Course.Faculty.Name))
+            .ForMember(csv => csv.Institute, o => o.MapFrom(p => p.Promoter.Institute.Name));
+        }
+
+        private string GetFullNameFor(Promoter promoter)
```

Comment:
`GetPromoterFullName` would be a better name.

Reply:
Done